octree_web_viewer/src/bin/points_web_viewer.rs

```diff
@@ -21,10 +21,15 @@ use std::sync::Arc;
 use structopt::StructOpt;
 
 /// HTTP web viewer for 3d points stored in OnDiskOctrees
+///
+/// Octrees that share a common path and (optional) identical subfolder structure can be served
+/// just by giving a full directory path, <common_octree_path>/<octree_identifier>/<optional_subfolder_suffix> as <DIR>.
+/// The option "--suffix_depth" allows to specify where the actual octree files reside in relation to the octree identifier.
 #[derive(StructOpt, Debug)]
 #[structopt(name = "points_web_viewer", about = "Visualizing points")]
 pub struct CommandLineArguments {
     /// The octree directory to serve, including a trailing slash.
+    /// this overrides <--prefix> and <--octree_id> options
```

Comment:
There is no more `--prefix` or `--octree_id`, right?

Reply:
exactly, this comment line has to go away